cps/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsThreadGroup.java

```diff
@@ -40,6 +40,8 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.Serializable;
+import java.nio.file.Files;
```

Comment:
Wouldn't this throw `NoClassDefFoundException` just on `CpsThreadGroup` instantiation on Java 6?